imageroot/bin/discover-smarthost

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (C) 2022 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+import sys
+import agent
+import os
+
+# Connect the local Redis replica. This is necessary to consistently start
+# the service if the leader node is not reachable:
+rdb = agent.redis_connect(use_replica=True)
+smtp_settings = agent.get_smarthost_settings(rdb)
+
+
+envfile = "smarthost.env"
+
+# Do not write the env file if smarthost is not configured
+if not smtp_settings['host']:
+    try:
+        os.unlink(envfile)
+    except:
+        pass
+    sys.exit(0)
```

Comment:
Since the file must be passed directly to podman, I think it must exists.

Reply:
done